Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -762,32 +778,36 @@ public static void setLogin(UUID uuid, String ip, long ipTime, JedisPool redisPo
         }
 
         // Redis
-        Redis.update(result, ipTime, redisPool, redisConfigNode);
+        Redis.update(result, ipTime, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode());
 
         // RabbitMQ
-        RabbitMQ.broadcast(result, ipTime, rabbitConnection);
+        try {
+            RabbitMQ.broadcast(result, ipTime, getRabbitConnection());
+        } catch (IOException | TimeoutException e) {
+            e.printStackTrace();
```

Comment:
Use the logger, Luke!

Reply:
This was actually just an edit of your own code, if I didn't use the logger its because you were not using it.